app/timetables/models.py

```diff
@@ -1,3 +1,10 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from app.shared.models import CommonInfo
+
+class MealOption(CommonInfo, models.Model):
```

Comment:
Since we are already inheriting `CommonInfo`, we don't need to inherit `models.Model` here.

Also could we clean up our migration files to one (after correcting the changes here) by squashing the two migration files using 
```
python manage.py squashmigrations timetables 0002
```
Check here for reference https://docs.djangoproject.com/en/1.9/topics/migrations/#squashing-migrations